baz/README.md

```diff
@@ -0,0 +1,34 @@
+# Baz
+
+## Overview
+Baz helps developers write unbreakable code by:
```

Comment:
I'd suggest changing `unbreakable` to `thoroughly validated`.

Reply:
It's the brand... https://baz.co